### DISCUSSION TITLE
Volume automatically created for clickhouse logs every time container starts up

### DISCUSSION THREAD

**Discussion by Shotster** [plausible/analytics#4013] [Self-Hosted Support] [answered]
Whenever the `plausible_events_db` container is restarted, a new volume is automatically created for the clickhouse DB warning and error logs...

```
2024-04-15 10:12:44 Logging warning to /var/log/clickhouse-server/clickhouse-server.log
2024-04-15 10:12:44 Logging errors to /var/log/clickhouse-server/clickhouse-server.err.log
```

While the installation seems to work fine, this behavior was "littering" my system with unused volumes. I resolved the issue by simply bind mounting a host directory to the container's `/var/log/clickhouse-server` directory, but I'm wondering if there's a better (recommended) way to handle this as part of the installation / setup process.

**Comment by ruslandoga** [comment]
👋 @Shotster 

I think we'll need to modify `docker-compose.yml` to explicitly mount these logs into a named volume. https://github.com/plausible/community-edition/issues/117

**Reply by ruslandoga** [reply, marked as answer]
PR: https://github.com/plausible/community-edition/pull/118

Is it similar to what you did? I'll try it out on my instance later today and if it's all fine, I'll merge it tomorrow.

**Reply by Shotster** [reply]
Yes, thank you!
